apps/sms/views/conversation/js/conversation.js

```diff
@@ -94,6 +94,10 @@ var ConversationView = {
 
   multiSimActionButton: null,
   init: function conv_init() {
+    const THROTTLE_OPTS = {
```

Comment:
OK, I was not sure it was important enough to be part of the conversationView's property. I'll move it there.